src/components/sidebar/sidebar.stories.js

```diff
@@ -113,6 +113,4 @@ storiesOf('Sidebar', module)
     }
   })
   .add(...makeStory('default', dlsThemeSelector))
-  .add(...makeStory('classic', classicThemeSelector, true))
-  .add(...makeButtonStory('with button', dlsThemeSelector))
-  .add(...makeButtonStory('with button classic', classicThemeSelector, true));
+  .add(...makeButtonStory('with button', dlsThemeSelector));
```

Comment:
I think that we could move the `dlsThemeSelector` directly to the `metadata`, we no longer need to pass it as an argument

Reply:
Resolved in 24df79c903de453e17595c2de56f7cabe2e3181d